shoop/core/cache/impl.py

```diff
@@ -72,7 +71,8 @@ def bump_version(self, cache_key):
         :type cache_key: str
         """
         namespace = _get_cache_key_namespace(cache_key)
-        version = str(time.time())
+        current_version = self.get_version(cache_key) or 1
+        version = current_version + 1
```

Comment:
Should the version still be string?

I.e. this would be something like:
 ```python
lastver = self.get_version(cache_key)
lastver_int = int(lastver) if isinstance(lastver, str) and lastver.isdigit() else 1
version = str(lastver_int + 1)
```